src/components/views/ViewMilestone.jsx

```diff
@@ -52,16 +55,18 @@ class ViewMilestone extends Component {
     feathersClient
       .service('milestones')
       .find({ query: { _id: milestoneId } })
-      .then(resp =>
+      .then(resp => {
+        BigNumber.config({ DECIMAL_PLACES: config.decimals })
```

Comment:
global config?

Reply:
Yeah, apparently the global config doesn't work so you need to set it again here..